int32 job_id
float64 x
float64 y
int32 zone_id
int32 assigned_agent_id # -1 if not yet assigned
bool completed
